Index: fonction_menu.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"Programme pour les fonctions menu(), LectureMessage(fichier,mesg) et SauvegardeMessage(fichier,mesg)\"\"\"\n\n\n\n#Notre dernière fonction est un menu, qui nous permet d'afficher des choix pour l'utilisateur\ndef menu():\n    print(f\"1. Lecture message\")\n    print(f\"2. Ecriture message\")\n    print(f\"3. *\")\n    print(f\"4. Quitter\")\n    a=int(input(f\"Faites un choix dans le menu\"))\n    if a=1:\n        LectureMessage(fichier,mesg)\n    elif a==2:\n        EcritureMessage(fichier,mesg)\n    elif a==3:\n        SauvegardeMessage(fichier,mesg)\n    elif a==4:\n        exit()\n    else:\n        print(\"Impossible ! Retour au menu !\")\n        menu()\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/fonction_menu.py b/fonction_menu.py
--- a/fonction_menu.py	(revision 33eb0df90397a92617b8d6713901ee18bb85082a)
+++ b/fonction_menu.py	(date 1645543103624)
@@ -1,22 +1,28 @@
 """Programme pour les fonctions menu(), LectureMessage(fichier,mesg) et SauvegardeMessage(fichier,mesg)"""
 
 
+# Fonction de lecture du message
+def LectureMessage(fichier, mesg):
+    # On ouvre d'abord le fichier message
+    with open("message.txt", "r") as fichier:
+        fichi
+
 
-#Notre dernière fonction est un menu, qui nous permet d'afficher des choix pour l'utilisateur
+# Notre dernière fonction est un menu, qui nous permet d'afficher des choix pour l'utilisateur
 def menu():
     print(f"1. Lecture message")
     print(f"2. Ecriture message")
-    print(f"3. *")
-    print(f"4. Quitter")
-    a=int(input(f"Faites un choix dans le menu"))
-    if a=1:
-        LectureMessage(fichier,mesg)
-    elif a==2:
-        EcritureMessage(fichier,mesg)
-    elif a==3:
-        SauvegardeMessage(fichier,mesg)
-    elif a==4:
+    print(f"4. *")
+    print(f"6. Quitter")
+    a = int(input(f"Faites un choix dans le menu"))
+    if a == 1:
+        LectureMessage(fichier, mesg)
+    elif a == 2:
+        EcritureMessage(fichier, mesg)
+    elif a == 4:
+        SauvegardeMessage(fichier, mesg)
+    elif a == 6:
         exit()
     else:
-        print("Impossible ! Retour au menu !")
+        print(f"Impossible ! Retour au menu !")
         menu()
